explainability-core/src/main/java/org/kie/trustyai/explainability/metrics/ConfusionMatrix.java

```diff
@@ -0,0 +1,37 @@
+package org.kie.trustyai.explainability.metrics;
+
+public class ConfusionMatrix {
```

Comment:
We can probably move this as nested class inside `FairnessMetrics` class. It looks like the usage is just as intermediate object used only internally.

Reply:
_(discussed elsewhere)_